assets/js/common.js

```diff
@@ -943,6 +943,19 @@ var hivepress = {
 			}
 		});
 
+		// Reset
+		hivepress.getComponent('reset').each(function () {
+			var field = $(this),
+				form = field.closest('form');
+
+			field.on('click', function(){
+				form.find('input:not([type="hidden"], [name="_category"]), select:not([type="hidden"])').each(function(){
```

Comment:
Please avoid hard-coding field names, it's ok to reset the whole form (there seems to be a reset() method or event for this in jQuery, I'm not sure how it works though - returns the old values or clears the whole form).